docs/user-guide/diffing.md

```diff
@@ -89,6 +89,17 @@ data:
     - kube-controller-manager
 ```
 
+It is possible to configure ignoreDifferences to be applied to all resources in every Application managed by an ArgoCD instance. In order to do so, resource customizations can be configured like in the example bellow:
+
+```yaml
+data:
+  resource.customizations.ignoreDifferences.all: |
```

Comment:
Please add `resource.customizations.ignoreDifferences.all` to https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/argocd-cm.yaml as well.

Reply:
Good point! Forgot about that file.
Pushed a new commit.